templates/NOTES.txt

```diff
@@ -1,3 +1,14 @@
 Please wait for several minutes for Harbor deployment to complete.
 Then you should be able to visit the Harbor portal at {{ .Values.externalURL }}. 
-For more details, please visit https://github.com/goharbor/harbor.
\ No newline at end of file
+For more details, please visit https://github.com/goharbor/harbor.
+
+{{ if .Values.userSettings }}
+If this is your first time Installing Harbor,
+You may need to use the Harbor Configuration API to
+finish your configuration by running:
+
+$ curl -i -X PUT -u "admin:{{ .Values.harborAdminPassword }}" \
```

Comment:
I like the idea of the hint, but revealing the admin password in output that could be going to a pipeline deployment is not a secure idea.